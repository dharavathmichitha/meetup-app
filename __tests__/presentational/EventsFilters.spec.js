import React from 'react';
import renderer from 'react-test-renderer';
import EventsFilters from '../../src/app/components/presentational/events/EventFilters';

beforeEach(() => {
  jest.resetModules();
});

describe('EventsFilters Component', () => {
  it('should match snapshot including default props', () => {
    const component = renderer.create(<EventsFilters />);

    const componentTree = component.toJSON();

    expect(componentTree).toMatchSnapshot();
  });
});
