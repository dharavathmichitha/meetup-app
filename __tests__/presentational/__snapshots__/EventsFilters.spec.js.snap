// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventsFilters Component should match snapshot including default props 1`] = `
<section
  className="filters"
>
  <input
    aria-label="Close"
    className="filters__event-search"
    id="searchinput"
    onChange={[Function]}
    placeholder="Search an event..."
    type="text"
  />
  <button
    className="filters__free-events btn btn"
    onClick={[Function]}
    type="button"
  >
    Discover Free Events
  </button>
</section>
`;
